### PR TITLE
use proper context import

### PR DESCRIPTION
**Pull request by poy**

### CODE REVIEW

**Comment by jba** [comment]
@jadekler Looks like I must not have write access, so my approval isn't sufficient.

File: storage/stiface/stiface_test.go [hunk 1]
```diff
@@ -21,7 +21,7 @@ import (
 	"os"
 	"testing"
 
-	"golang.org/x/net/context"
+	"context"
```
Comment (jadekler) [inline storage/stiface/stiface_test.go]: Could you move this up? Ditto elsewhere.

Comment (poy) [reply storage/stiface/stiface_test.go]: Done. Forgot that `goimports` doesn't mess with groups.

File: storage/stiface/interfaces.go [hunk 2]
```diff
@@ -33,9 +33,10 @@ package stiface
 import (
 	"io"
 
+	"context"
```
Comment (jadekler) [inline storage/stiface/interfaces.go]: (here too please :) )

**Comment by poy** [comment]
@jadekler PTAL

**Comment by jadekler** [comment]
Thanks!

**Pull request merged**